docs/development/extensions-core/datasketches-theta.md

```diff
@@ -23,7 +23,7 @@ title: "DataSketches Theta Sketch module"
   -->
 
 
-This module provides Apache Druid aggregators based on Theta sketch from [datasketches](https://datasketches.github.io/) library. Note that sketch algorithms are approximate; see details in the "Accuracy" section of the datasketches doc.
+This module provides Apache Druid aggregators based on Theta sketch from [datasketches](https://datasketches.apache.org/) library. Note that sketch algorithms are approximate; see details in the "Accuracy" section of the datasketches doc.
```

Comment:
It looks like there are some other docs that also point to https://datasketches.github.io/, should they all be updated?

* datasketches-hll.md
* datasketches-tuple.md
* datasketches-extension.md
* datasketches-quantiles.md

Reply:
Yes! Please!  Can you do it or do you want me to submit PRs ?

Reply:
Ah, I can do it, sorry forgot about this :+1: